### PR TITLE
Update user documentation for Hybrid Scan threshold configs

### PR DESCRIPTION
**Pull request by sezruby**
### What is the context for this pull request?

 - **Tracking Issue**: n/a
 - **Parent Issue**: n/a
 - **Dependencies**: n/a
 - 
### What changes were proposed in this pull request?

Use documentation update for #300.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

tested by github page

### CODE REVIEW

**Comment by sezruby** [comment]
@apoorvedave1 @thugsatbay Could you review the document asap? as we will release this week. Thanks!

File: docs/_docs/02-ug-configuration.md [hunk 1]
```diff
@@ -17,6 +17,6 @@ classes: wide
...
-| spark.hyperspace.index.hybridscan.delete.enabled   | false                                                                                    | (temporary) Enable Hybrid Scan for deleted files. Lineage is required to handle deleted files in Hybrid Scan.  | 0.3.0         |
-| spark.hyperspace.index.hybridscan.delete.maxNumDeleted   | 30                                                                                    | Threshold of the maximum number of deleted files for Hybrid Scan. | 0.3.0         |
+| spark.hyperspace.index.hybridscan.maxAppendedRatio   | 0.3                                                                          |  Ratio threshold for the max. amount of appended data for Hybrid Scan. If there's more appended data than this threshold, Hybrid Scan won't be applied. It's 30% (0.3) by default.   | 0.4.0         |
```
Comment (apoorvedave1) [inline docs/_docs/02-ug-configuration.md:20]: could you please clarify "amount of appended data"? is it number of files? Is it file size?

File: docs/_docs/02-ug-configuration.md [hunk 2]
```diff
@@ -17,6 +17,6 @@ classes: wide
...
+| spark.hyperspace.index.hybridscan.maxDeletedRatio   | 0.2                                                                            | Ratio threshold for the max. amount of deleted data for Hybrid Scan. If there's more deleted data than this threshold, Hybrid Scan won't be applied. It's 20% (0.2) by default. Lineage is required to handle deleted files with Hybrid Scan.   | 0.4.0         |
```
Comment (apoorvedave1) [inline docs/_docs/02-ug-configuration.md:21]: could you please clarify "amount of appended data"? is it number of files? Is it file size?

File: docs/_docs/03-ug-mutable-dataset.md [hunk 3]
```diff
@@ -199,35 +197,35 @@ query.show
...
 
 You can use the following configurations to enable Hybrid Scan for indexes on a dataset with both append and delete files.
 
-We currently provide one threshold config:
-`spark.hyperspace.index.hybridscan.delete.maxNumDeleted`. If there are more deleted files than the config value,
-we do not perform Hybrid Scan for the index.
+We currently provide one threshold config for deleted files:
+`spark.hyperspace.index.hybridscan.maxDeletedRatio`
+
+It indicates the maximum ratio of deleted data compared to index data to perform Hybrid Scan.
```
Comment (apoorvedave1) [inline docs/_docs/03-ug-mutable-dataset.md:215]: throughout the doc could you please add the info what is the ration about?
`maximum ratio of deleted data compared to index data`
Is it data file size? Is it num files?

**Review by apoorvedave1 — DISMISSED** [review]
minor comments but otherwise LGTM, thanks @sezruby

**Review by imback82 — APPROVED** [review]
LGTM, thanks @sezruby!

**Pull request merged**